okio/src/commonTest/kotlin/okio/AbstractFileSystemTest.kt

```diff
@@ -58,8 +58,8 @@ abstract class AbstractFileSystemTest(
     val cwdString = cwd.toString()
     assertTrue(cwdString) {
       cwdString.endsWith("okio${Path.DIRECTORY_SEPARATOR}okio") ||
-        cwdString.endsWith("${Path.DIRECTORY_SEPARATOR}okio-parent-okio-jsLegacy-test") ||
-        cwdString.endsWith("${Path.DIRECTORY_SEPARATOR}okio-parent-okio-jsIr-test") ||
```

Comment:
@swankjesse I've seen you changing these from `js-ir` to `jsIr` in a recent PR, but I couldn't figure out what these actually come from. It seems to have changed in some stdlib shipped with 1.5.0. Do advise if you think this is unexpected!